src/interface/src/app/treatments/treatments.state.ts

```diff
@@ -121,6 +121,15 @@ export class TreatmentsState {
       );
   }
 
+  // TODO: allow us to set other values, besides name?
```

Comment:
I think this might be the only value we want to patch/update directly, or at least, anytime soon?

Reply:
you can do this by changing `name:string` to `treatmentPlan: Partial<TreatmentPlan>`